contrib/completion/bash/sdk

```diff
@@ -49,7 +49,7 @@ __sdkman_complete_command() {
 			;;
 	esac
 
-	read -r -a COMPREPLY < <(compgen -W "${candidates[*]}" -- "$current_word")
+	mapfile -t COMPREPLY < <(compgen -W "${candidates[*]}" -- "$current_word")
```

Comment:
Hi, we are deliberately not using process substitution, as more often than not users run SDKMAN! in Bash POSIX mode, which among other things disables process substitution (that's probably why some tests fail, as they are ran with POSIX mode enabled).

Reply:
Thank-you for the info. It was not straightforward at all because Bash5.1 accepts the constructs in posix mode. I found alternative method to plase old Bash posix mode with mapfile array from command output.